code/modules/mining/mine_turfs.dm

```diff
@@ -1112,6 +1114,29 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 		/mob/living/simple_animal/hostile/asteroid/pillow = 2
 	)
 	var/sanity = 1
+	var/turf/floor_type = /turf/unsimulated/floor/asteroid
+
+/turf/unsimulated/floor/asteroid/cave/permafrost
+	mob_spawn_list = list(
+		/mob/living/simple_animal/hostile/bear = 4,
+		/mob/living/simple_animal/hostile/asteroid/pillow = 3,
+		/mob/living/simple_animal/hostile/scarybat = 5,
+		/mob/living/simple_animal/hostile/giant_spider/hunter = 4,
+		/mob/living/simple_animal/hostile/giant_spider/nurse = 3,
```

Comment:
spider migration redux

Reply:
They're in rock, a lot of rock.

Also, we need more snow mobs

Reply:
That can be arranged